### PR TITLE
Feature/add template

### PR DESCRIPTION
**Pull request by Clyde21c**
## PR Description

[//]: # Add repo template

## Related Issues

[//]: # (Please link the related issues, if applicable.)

## Checklist

- [x] **Assignees** ran the [make format](https://github.com/dongminlee94/meta-rl#development).
- [x] **Assignees** ran the [make lint](https://github.com/dongminlee94/meta-rl#development) command and the code passed the linters.
- [x] **Assignees** updated the README, if necessary.
- [x] **Assignees** checked the runnability of the code.
- [x] **Reviewers** checked the runnability of the code.

## Optional section (e.g., code usage, experimental results, TODO)

[//]: # (If Assignees have any additional information, please add them.)

### CODE REVIEW

**Comment by sujeongim** [comment]
test

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (sujeongim) [inline LICENSE:1]: test

**Review by sujeongim — COMMENTED** [review]
comment

File: LICENSE (hunk 1 above)
Comment (Clyde21c) [reply LICENSE:1]: comment 확인했습니다

**Comment by sujeongim** [comment]
LGTM

**Review by sujeongim — APPROVED** [review]
LGTM

**Pull request merged**